### PR TITLE
added linter and updated all outdated packages

### PR DESCRIPTION
**Pull request by ccades**

### CODE REVIEW

File: database/Stock.js [hunk 1]
```diff
@@ -4,40 +4,38 @@ const Schema = require('./index.js');
...
-    required: [true, 'currentPrice field is required']
+    required: [true, 'currentPrice field is required'],
   },
   marketCap: {
     type: Number,
   },
   sector: {
     type: String,
-  }
+  },
```
Comment (morgangrace11) [inline database/Stock.js:25]: Does this comma need to be there?

**Review by morgangrace11 — COMMENTED** [review]
lgtm! All the changes are great, a nice uniformity with the const vars and spacing!

File: .eslintrc.js [hunk 2]
```diff
@@ -1,10 +1,3 @@
...
- * These rules enforce the Hack Reactor Style Guide
- *
- * Visit this repo for more information:
- *   https://github.com/reactorcore/eslint-config-hackreactor
- */
-
 module.exports = {
-  extends: './node_modules/eslint-config-hackreactor/index.js'
+    "extends": "airbnb",
 };
```
Comment (therobinkim) [inline .eslintrc.js:3]: put a newline at the end of your files

File: database/Stock.js (hunk 1 above)
Comment (therobinkim) [reply database/Stock.js:25]: I think AirBnb likes the trailing commas

File: server/index.js [hunk 3]
```diff
@@ -1,41 +1,40 @@
 const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
-const db = require('../database/index.js')
-const transactions = require('../database/Transaction.js')
-const stocks = require('../database/Stock.js')
+const db = require('../database/index.js');
+const transactions = require('../database/Transaction.js');
```
Comment (therobinkim) [inline server/index.js:5]: seems odd to have capitalized file names, esp on the backend

**Comment by therobinkim** [comment]
🎉

**Pull request merged**